pkg/connector/capabilities.go

```diff
@@ -148,6 +148,14 @@ var whatsappCaps = &event.RoomFeatures{
 	ReactionCount:       1,
 	ReadReceipts:        true,
 	TypingNotifications: true,
+	DisappearingTimer: &event.DisappearingTimerCapability{
+		Types: []string{string(event.DisappearingTypeAfterSend)},
```

Comment:
When changing capabilities, `capID` and `GetBridgeInfoVersion` also need to be updated so it's automatically updated in all rooms